pkg/operator2/operator.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"github.com/openshift/library-go/pkg/authentication/bootstrapauthenticator"
```

Comment:
move among other openshift imports

Reply:
Not sure why `go fmt` did not work previously. Fixed it now.